cuebot/src/main/resources/conf/ddl/postgres/migrations/V21__AddShowStats.sql

```diff
@@ -0,0 +1,33 @@
+CREATE TABLE show_stats (
```

Comment:
The file is named `V21` but based on what's in master it should be `V16` I think.

Reply:
I think we have discussed this on the past, but maybe I'm remembering it wrong. 
Merging migration changes is being a pain as there is no gap between migrations to simplify handling conflicts. 
If internally we develop a change that adds v16, v17 and v18 for example and on the open source end someone develops a v16, it would mean having to push all of our changes ahead or even reordering.

To avoid this, giving a gap between migrations makes it easier to conciliate conflicts.

Reply:
File renamed to V16 as discussed on the TSC